README.md

```diff
@@ -40,6 +40,17 @@ glslang is a GLSL and HLSL to SPIR-V compiler, usable both as a command-line too
 * Contact: David Neto <dneto@google.com>
 * Compiler requirements: C++11 compatible compiler (MSVC 2013+ on Windows)
 
+## DXC
+DirectXShaderCompiler (DXC) is Microsoft's next-gen official HLSL compiler, based on LLVM/Clang.
+It can also compile HLSL into SPIR-V, contributed by Google.
```

Comment:
Sentence structure feels weird. Maybe "thanks to contribution from Google".

"also" feels superfluous as it was not previously specified it is primarily\originaly HLSL->DXIL compiler.

Reply:
Done.